backend/bookings/services/bookingService.js

```diff
@@ -1,23 +1,28 @@
-const { sequelize } = require("@utopia-airlines-wss/common/db");
-const { Booking } = require("@utopia-airlines-wss/common/models");
-const {  NotFoundError,  handleMutationError   } = require("@utopia-airlines-wss/common/errors");
+const {sequelize} = require("@utopia-airlines-wss/common/db");
+const {Booking} = require("@utopia-airlines-wss/common/models");
+const {NotFoundError, handleMutationError} = require("@utopia-airlines-wss/common/errors");
 
 const findBookingById = async (id, options) => {
   const booking = await Booking.findByPk(id, options);
   if (!booking) throw new NotFoundError("cannot find booking");
   return booking;
 };
 
-const replacer = (key, value) =>  key === "passengers"
+const replacer = (key, value) => key === "passengers"
   ? value.map((passenger) => passenger.toJSON("full"))
   : value;
 
-
 const bookingService = {
-  async findAllBookings({ isActive = true } = {}) {
-    const where = { isActive };
-    const bookings = await Booking.findAll({
+  async findAllBookings(
+    {isActive = true, offset = 0, limit = 10} = {}) {
+    const where = {isActive};
+    limit = limit ? Math.min(+limit, 40) : 10;
```

Comment:
Is there a reason that you are modifying the limit variable?
I think it would be clearer if you just set the property in the config object to this expression like so.
```javascript
const bookings = await Booking.findAndCountAll({
      where,
      limit: limit ? Math.min(+limit, 40) : 10,
      offset: offset ?? 0,
      distinct: true,
      include: [
            ...
      ],
});

Reply:
Hmm, It is definitely more concise to set the limit and offset in the config variable.